src/index.ts

```diff
@@ -302,3 +302,7 @@ export {
   type ChatBotMessage,
   GPTVersion,
 } from './chatbot/chatbot.js';
+
+export { isEmail } from './validation/isEmail.js';
+export { isFQDN } from './validation/isFQDN.js';
```

Comment:
What is it? Maybe use the full name?

Reply:
`FQDN` stands for Fully Qualified Domain Name. Not sure I should use the full ? What do you think ?

Reply:
I think it's okay to let it like that, this acronym is known enough in my sense. It'd be like using full name for HTTP.